pyproject.toml

```diff
@@ -54,6 +55,9 @@ dbt-all = [
     "dbt-snowflake",
     "dbt-spark",
 ]
+dbt-athena-community = [
```

Comment:
We can probably name it `dbt-athena`, from a Cosmos perspective it probably is not significant if a dbt plugin is community or not.

Reply:
Done -- let me know if my change is correct

(I've tested this locally, just need to be careful here to not install the outdated adapter https://pypi.org/project/dbt-athena/)